src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/services/GetPersonService.kt

```diff
@@ -12,15 +12,17 @@ class GetPersonService(
   @Autowired val probationOffenderSearchGateway: ProbationOffenderSearchGateway,
 ) {
   fun execute(pncId: String): Map<String, Person?>? {
-    val personFromPrisonerOffenderSearch = prisonerOffenderSearchGateway.getPersons(pncId = pncId)
+    val personsFromPrisonerOffenderSearch = prisonerOffenderSearchGateway.getPersons(pncId = pncId)
     val personFromProbationOffenderSearch = probationOffenderSearchGateway.getPerson(pncId)
 
-    if (personFromPrisonerOffenderSearch.isEmpty() && personFromProbationOffenderSearch == null) {
+    if (personsFromPrisonerOffenderSearch.isEmpty() && personFromProbationOffenderSearch == null) {
       return null
     }
 
+    val personFromPrisonerOffenderSearch = if (personsFromPrisonerOffenderSearch.isEmpty()) null else personsFromPrisonerOffenderSearch.first()
+
     return mapOf(
-      "prisonerOffenderSearch" to personFromPrisonerOffenderSearch.first(),
+      "prisonerOffenderSearch" to personFromPrisonerOffenderSearch,
```

Comment:
```suggestion
      "prisonerOffenderSearch" to personFromPrisonerOffenderSearch.firstOrNull(),
```

Could we use `firstOrNull()` instead? Looks like it does the same thing as the if statement above.

![image](https://user-images.githubusercontent.com/42817036/228551492-ae33ac62-0039-479f-8107-47bfb5dee5d5.png)